apps/web/next.config.js

```diff
@@ -185,6 +198,16 @@ const nextConfig = {
         source: "/embed/embed.js",
         destination: process.env.NEXT_PUBLIC_EMBED_LIB_URL?,
       }, */
+      {
+        source: `/:user((?!${pages.join("|")}))/:type`,
```

Comment:
Made a similar approach to our website. This will rewrite everything that is not a static route (AKA [user])